### DISCUSSION TITLE
Flutter architecture for app

### DISCUSSION THREAD

**Discussion by addiraw** [IICDC/Introductory#3] [General]
[flutter-apps-architecture-styling-conventions-state-management](https://dev.to/noureldinshobier/building-scalable-flutter-apps-architecture-styling-conventions-state-management-40c9)

**Comment by rajput-abdullah** [comment]
Flutter Official Architecture Docs:
https://docs.flutter.dev/resources/architectural-overview

**Reply by addiraw** [reply]
It's flutter architecture docs right. 
What's the general architecture for building applications in flutter ?

**Comment by rajput-abdullah** [comment]
Yes, this is Flutter architecture docs.
Generally, most of the people uses MVP or MVC.
